index.yaml

```diff
@@ -1,6 +1,28 @@
 apiVersion: v1
 entries:
   admission-controller:
+  - apiVersion: v2
+    appVersion: 0.0.1
+    created: "2021-11-16T21:24:51.096231-08:00"
+    dependencies:
+    - condition: proxy-scanner.enabled
+      name: proxy-scanner
+      repository: https://lacework.github.io/helm-charts
+      version: 0.2.2
+    description: Lacework admission controller using Lacework proxy scanner
+    digest: 04e1c54725cfb4ef511957e3684dc5318fa3439e9095c89e3da3b0c16f75f3a6
+    keywords:
+    - admission-controller
+    - runtime-security
+    - lacework
+    maintainers:
+    - email: chitra.kumar@lacework.net
+      name: chitrapkumar
+    name: admission-controller
+    type: application
+    urls:
+    - admission-controller-0.2.0.tgz
+    version: 0.2.0
```

Comment:
Once you make that change, this would change too. You you have to push a new tag to docker hub as well.